src/buildApplication/generateFunctionsMap.ts

```diff
@@ -85,21 +84,25 @@ async function processDirectoryRecursively(
 	};
 }
 
+type FunctionConfig = {
+	runtime: 'edge';
```

Comment:
I don't think this is right, we want to have the `edge` runtime but this can also be `nodejs`, and we do check for that in the code:
```
if (functionConfig?.runtime !== 'edge') {
```
(I'm actually surprised that typescript is not complaining here 😕)

(anyways what I am saying is that I think that your type is a bit misleading and for example makes that if statement look incorrect since you're saying the the runtime is always `edge`)

```suggestion
	runtime: 'edge'|'nodejs';
```

Reply:
Hey man, I just copied the type that was already used and added the additional properties to it for future usage, haha. Fixed.

There wouldn't be a listing complaint since runtime can technically be undefined if functionConfig is null, so it would still be possible to not equal 'edge'.

Reply:
> There wouldn't be a listing complaint since runtime can technically be undefined if functionConfig is null, so it would still be possible to not equal 'edge'.

ahhh yeah, good point 😁